packages/e2e-tests/specs/editor/blocks/image.test.js

```diff
@@ -34,19 +35,23 @@ async function upload( selector ) {
 	const filename = uuid();
 	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
 	fs.copyFileSync( testImagePath, tmpFileName );
-	await inputElement.uploadFile( tmpFileName );
+	await inputElement.setInputFiles( tmpFileName );
 	return filename;
 }
 
 async function waitForImage( filename ) {
-	await page.waitForSelector(
-		`.wp-block-image img[src$="${ filename }.png"]`
+	const imageElement = await page.waitForSelector(
+		`.wp-block-image img[src$="${ filename }.png"]`,
+		{ state: 'visible' }
```

Comment:
FYI that[ I think that state for this will default to visible](https://playwright.dev/docs/api/class-page#page-wait-for-selector)! You use this a few times throughout. But, this does read very nicely if you want to emphasize the importance of it being visible :)

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/34089/commits/9b34909f703ad15ad6a002e4cb4d3f6b52cf5c79